products/products.js

```diff
@@ -0,0 +1,10 @@
+import wines from '../data/wines.js';
+import renderWines from './render-wines.js';
+
+const list = document.getElementById('wines');
```

Comment:
the wine list is not rendering.  I don't see a "wines" ID on your html for this line grab.